modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/components/PartialResults.tsx

```diff
@@ -73,11 +73,11 @@ const PartialResults: React.FC<IProps> = ({reportDataURL}) => {
 								{totalItems === 1
 									? Liferay.Util.sub(
 											Liferay.Language.get('x-entry'),
-											[totalItems]
+											totalItems
```

Comment:
These changes don't look right, as we don't want to change how the `sub(` API works